packages/accessibility/src/behaviors/Accordion/accordionTitleBehavior.ts

```diff
@@ -49,6 +49,8 @@ type AccordionTitleBehaviorProps = {
   active?: boolean
   /** If at least one panel needs to stay active and this title does not correspond to the last active one. */
   canBeCollapsed?: boolean
+  /** Disabled */
+  disabled?: boolean
```

Comment:
Would remove this comment, or provide some more meaningful, like

/** An accordion title can show it is currently unable to be interacted with. */